bin/offline-dbg

```diff
@@ -162,19 +162,40 @@ do_collect-sos() {
     local sos=$1
 
     local dir_name=`tar -tf $sos | head -1 | cut -f1 -d"/"`
-    sos_files="$dir_name/sos_commands/rpm
-    $dir_name/sos_commands/openvswitch
-    $dir_name/var/lib/openvswitch
-    $dir_name/etc/openvswitch
-    $dir_name/usr/local/etc/openvswitch"
+
+    sos_files="${dir_name}/sos_commands/rpm
```

Comment:
I would avoid adding newlines (`\n`) in a bash variable that will then be used in a commandline (without xargs)